tests/benchmarks/exercism_practice.py

```diff
@@ -177,7 +178,7 @@ def run_exercise_sync(problem_dir, language="python", max_iterations=2):
     return result
 
 
-def summarize_results(results):
+def tqdm_summary(results):
```

Comment:
I renamed this to make it clear what it's purpose was and how it is distinct from the other summary method.